dsms/knowledge/properties/apps.py

```diff
@@ -101,20 +100,20 @@ def run(
                 (not in the background), but the object should wait until the job finished.
                 Warning: this may lead to a request timeout for long running jobs!
                     Job details may not be associated anymore when this occurs.
-            token (bool, optional): Whether the job also should receive the access
-                 token as paramter.If `True`, the JWT will be set as parameter with
-                 the name ´access_token`.
+            use_sdk (bool, optional): Whether the job is using the SDK internally
+                and hence should receive parameters from the current the SDK session.
+                If set to True, the `token`, `host_url`, `ping_dsms`, `verify_ssl`,
+                `request_timeout`, `encoding` and `kitem_repo` will be set as parameters.
+                Defaults to False.
```

Comment:
I think the `use_sdk` name is not self-descritptive enough . The description is also not clear to me.

Reply:
If an app or pipeline is using the SDK internally (e.g the data2rdf), you might want to pass some of the SDK parameters such as the host url, ssl verification, etc. to the app itself as a general setting or during runtime.

These are basically the same parameters which you are using in the current SDK session and the `use_sdk` is a shorthand for passing the most important sdk configuration fields as parameters to the app.

You are able to pass this either:

* to the app config
* when you launch the app job

Reply:
I understand now better. Here are some suggestions for the name:

- pass_sdk_params_to_app
- expose_sdk_config
- forward_sdk_config_to_app
- app_can_use_sdk_config
- propagate_sdk_params
- enable_param_pass_through
- allow_param_propagation

A suggestion for the doc string:

```
Determines whether SDK parameters (such as host URL, SSL verification, etc.) should be passed through or propagated to the app using the SDK.

If set to True, the SDK's configuration will be made available for the app to use, allowing it to inherit settings such as the host URL or SSL configuration. If False, the app will not have access to these parameters, and the SDK will handle its own configuration independently.

:param propagate_sdk_params: (bool) Controls whether SDK parameters are passed through or propagated to the app.
```

Reply:
I like `expose_sdk_config` the most :)

Reply:
Done.